x/claim/keeper/hooks.go

```diff
@@ -51,7 +51,7 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochInfo epochstypes.EpochInfo)
 	if strings.HasPrefix(epochInfo.Identifier, "airdrop-") {
 		airdropIdentifier := strings.TrimPrefix(epochInfo.Identifier, "airdrop-")
 		airdropFound := k.GetAirdropByIdentifier(ctx, airdropIdentifier)
```

Comment:
```suggestion
		airdrop := k.GetAirdropByIdentifier(ctx, airdropIdentifier)
```
this might be slightly clearer?

Reply:
definitely!